10.rb

```diff
@@ -0,0 +1,14 @@
+def seperate_even_odd(arr)
+    odd = []
+    even = []
+    result = []
+    arr.each do
+        |i|
+        odd.push(i) if i%2 != 0
+        even.push(i) if i%2 == 0
```

Comment:
Prefer ruby's built-in method
```suggestion
        odd.push(i) if i.odd?
        even.push(i) if i.even?
```

Reply:
Since a number is either odd or even
```suggestion
  i.odd? ? odd.push(i) : even.push(i)
```